content/projects/morse-code/_index.md

```diff
@@ -59,7 +59,8 @@ Your directory structure should look like this:
 
 ## Instructions
 
-Create two functions:
+- Create an object of that contains morsecode. See resources below for examples. Make sure that you create only one object.
```

Comment:
The first sentence on the first point doesn't seem right. Shouldn't it be `Create an object that contains morsecode characters`?